.gitignore

```diff
@@ -1,6 +1,7 @@
 .build_cache/
 build/
 dist/
+lib/*.jl
```

Comment:
Interesting. Are there typically Julia source files in lib? I thought these usually live within src instead.

Reply:
Babel copied from src, these jl files should never have made it there :)